templates/quarkus-camel-api-consumer/skeleton/pom.xml

```diff
@@ -228,22 +229,22 @@
                 </executions>
             </plugin>
 
+            {%- if values.service_registry_provider_url %}
             <!--
             This plugin connects with the Service Registry and download
             the contracts (OpenApi) used in the project -->
-            {%- if values.service_registry_provider_url %}
             <plugin>
                 <groupId>io.apicurio</groupId>
                 <artifactId>apicurio-registry-maven-plugin</artifactId>
-                <version>2.6.4.Final</version>
+                <version>${registry.version}</version>
                 <executions>
                   <execution>
                     <phase>generate-sources</phase>
                     <goals>
                       <goal>download</goal> 
                     </goals>
                     <configuration>
-                      <registryUrl>${{ values.service_registry_provider_url }}</registryUrl> 
+                      <registryUrl>${{ values.service_registry_provider_url }}/api</registryUrl> 
```

Comment:
I've seen apicurio-registry-maven-plugin work with a bare hostname URL (eg, `http://apicurio-registry.namespace.svc`) and with a path to `/apis/registry/v2` (eg, `http://apicurio-registry.namespace.svc/apis/registry/v2`). Is there a reason to add `/api` here, where the base provider url wouldn't work?

Reply:
Most probably apicurio handle it, I'll remove it and try again, I took the url with /api from documentation https://www.apicur.io/registry/docs/apicurio-registry/1.3.3.Final/getting-started/assembly-managing-registry-artifacts-maven.html#managing-artifacts-using-maven-plugin